manage.py

```diff
@@ -629,10 +690,16 @@ def field_lists(name, tables_only=False):
 
     start = time()
 
+    # contract_summary and award_summery field lists can not be run at the same time as they cause deadlocks
```

Comment:
To make this a little cleaner we can have a simple dependency graph similar to `summary_tables`

Reply:
I have looked into this and requirs a fair amount more code then what is there currently, especially as this is just for a single dependancy.  If it were any more compliacted a dependancy graph I would agree.

I also can not see clearly how to share the logic with `summary_tables` as single function as the `submit` and `run` closures are specialised to the summary case, along with the concept of 'initial', 'middle' and 'end'.  So from initial look at the refactor I think it would be better as some kind of class with overriddable methods and I worry about too much abstraction making the code difficult to reason about.  It would also take me a fair amount of time to do well.

Reply:
Okay, let's go as-is.

Reply:
I had a look at it and extracted a method fairly straight-forwardly.

Reply:
looks great.  I think it actully makes the `summary_tables` code more easy to understand.  Not sure I would have managed that.